leapp/reporting/__init__.py

```diff
@@ -269,15 +269,28 @@ def _sanitize_entries(entries):
         entries.append(Audience('sysadmin'))
 
 
-def create_report(entries):
-    """
-    Create final report message
-    """
-
+def _create_report_object(entries):
     report = {}
 
     _sanitize_entries(entries)
     for entry in entries:
         entry.apply(report)
 
-    produce(Report(report=report))
+    return Report(report=report)
+
+
+def create_report(entries):
+    """
+    Produce final report message
+    """
+    produce(_create_report_object(entries))
+
+
+def create_report_from_error(error_dict):
+    """
+    Convert error json to report json
+    """
+    error = ErrorModel.create(error_dict)
+    entries = [Title(error.message), Summary(error.details or ""), Severity('high'), Audience('sysadmin')]
```

Comment:
Even though Summary itself is older code - why is it Summary here and Detail in the error display at the end of a leapp run?

Reply:
Not to mention that the dict labeled as `Detail` often contains a `details` key.